Patch 1: magic

Adkorh opened the pull request.

Bandlou committed:
Magic: update 1

Bandlou committed:
Update The Praeter.md

Bandlou committed:
Update The Praeter.md

Bandlou committed:
Update The Praeter.md

Bandlou committed:
Update The Praeter.md

Bandlou committed:
Update The Praeter.md

Bandlou committed:
Update The Praeter.md

Bandlou committed:
Update The Praeter.md

Bandlou committed:
Update The Praeter.md

Bandlou committed:
Update The Praeter.md

Bandlou committed:
Update The Praeter.md

Bandlou committed:
Update The Praeter.md

Bandlou committed:
Create Schools.md

Bandlou committed:
Rename Schools.md to The Schools of Magic.md

Bandlou committed:
Update The Schools of Magic.md

Bandlou committed:
Update The Schools of Magic.md

Bandlou committed:
Update The Schools of Magic.md

Bandlou committed:
Update The Schools of Magic.md

Bandlou committed:
Create The Art of Magic.md

Bandlou committed:
Update The Art of Magic.md

Bandlou committed:
Update The Art of Magic.md

Bandlou committed:
Update The Art of Magic.md

Bandlou committed:
Update The Schools of Magic.md

Bandlou committed:
Update The Praeter.md

Bandlou committed:
Update The Art of Magic.md

Bandlou committed:
Update The Schools of Magic.md

Bandlou committed:
Update The Schools of Magic.md

Bandlou committed:
Update The Schools of Magic.md

Bandlou committed:
Update The Schools of Magic.md

Bandlou committed:
Update The Schools of Magic.md

Bandlou committed:
Update The Schools of Magic.md

Bandlou committed:
Update The Schools of Magic.md

Bandlou committed:
Update The Schools of Magic.md

Bandlou committed:
Update The Schools of Magic.md

cela a comme base cette idée

Bandlou committed:
Update The Art of Magic.md

Bandlou committed:
Update and rename The Schools of Magic.md to The Frame of Magic.md

Bandlou committed:
Update The Frame of Magic.md

Bandlou committed:
Update The Frame of Magic.md

Bandlou committed:
Update The Frame of Magic.md

Bandlou committed:
Update The Frame of Magic.md

Bandlou committed:
Update The Frame of Magic.md

Bandlou committed:
Update The Frame of Magic.md

Bandlou committed:
Update The Frame of Magic.md

Bandlou committed:
Update The Frame of Magic.md

Bandlou committed:
Update The Frame of Magic.md

Bandlou committed:
Update The Frame of Magic.md

Bandlou committed:
Update The Art of Magic.md

Bandlou committed:
Update The Frame of Magic.md

Bandlou committed:
Update The Frame of Magic.md

Bandlou committed:
Update The Frame of Magic.md

Bandlou committed:
Update The Frame of Magic.md

Bandlou committed:
Update The Frame of Magic.md

Bandlou committed:
Update The Frame of Magic.md

Bandlou committed:
Update and rename The Praeter.md to 1. The Praeter.md

Bandlou committed:
Update and rename The Art of Magic.md to 2. The Art of Magic.md

Bandlou committed:
Update and rename The Frame of Magic.md to 3. The Frame of Magic.md

Bandlou committed:
Update 3. The Frame of Magic.md

Bandlou committed:
Update 2. The Art of Magic.md

Bandlou committed:
Update 2. The Art of Magic.md

Bandlou committed:
Update 2. The Art of Magic.md

Bandlou committed:
Update 2. The Art of Magic.md

Bandlou committed:
Create Disciples of Brynjar

Bandlou committed:
Rename Disciples of Brynjar to Disciples of Brynjar.md

Bandlou committed:
Update Disciples of Brynjar.md

Bandlou committed:
Update and rename Disciples of Brynjar.md to Brynjards.md

Bandlou committed:
Update Brynjards.md

Bandlou committed:
Update 1. The Praeter.md

Bandlou committed:
Update 1. The Praeter.md

Bandlou committed:
Update 1. The Praeter.md

Bandlou committed:
Update 1. The Praeter.md

Bandlou committed:
Update 1. The Praeter.md

Bandlou committed:
Update and rename Brynjards.md to Treaty on Magic.md

Bandlou committed:
Update Treaty on Magic.md

Bandlou committed:
Rename Treaty on Magic.md to The Treaty on Magic.md

Bandlou committed:
Create The Codex.md

Bandlou committed:
Update The Codex.md

Bandlou committed:
Update The Codex.md

Bandlou committed:
Update The Codex.md

Bandlou committed:
Update The Codex.md

Bandlou committed:
Update The Treaty on Magic.md

Bandlou committed:
Update The Treaty on Magic.md

Bandlou committed:
Update The Treaty on Magic.md

Bandlou committed:
Update The Treaty on Magic.md

Bandlou committed:
Update The Treaty on Magic.md

Bandlou committed:
Update The Codex.md

Bandlou committed:
Update The Codex.md

Bandlou committed:
Update The Treaty on Magic.md

Bandlou committed:
Update The Codex.md

Bandlou committed:
Update The Codex.md

Bandlou committed:
Update The Codex.md

Bandlou committed:
Update The Codex.md

Bandlou committed:
Update The Codex.md

Bandlou committed:
Update The Codex.md

Bandlou committed:
Update The Codex.md

Bandlou committed:
Update and rename The Codex.md to The Codex of Nendal.md

Bandlou committed:
Update and rename The Codex of Nendal.md to The Nendal Codex.md

Bandlou committed:
Update The Nendal Codex.md

Bandlou committed:
Update The Nendal Codex.md

Bandlou committed:
Update The Nendal Codex.md

Bandlou committed:
Update The Nendal Codex.md

Bandlou committed:
Update The Nendal Codex.md

Bandlou committed:
Update The Nendal Codex.md

Bandlou committed:
Update The Treaty on Magic.md

Bandlou committed:
Update The Nendal Codex.md

Bandlou committed:
Update The Nendal Codex.md

Bandlou committed:
Update The Nendal Codex.md

Bandlou committed:
Update The Nendal Codex.md

Bandlou committed:
Update The Nendal Codex.md

Bandlou committed:
Update The Treaty on Magic.md

Bandlou committed:
Update The Treaty on Magic.md

kestal commented on Magic/2. The Art of Magic.md:
```diff
@@ -0,0 +1,32 @@
+- Magical beasts: these creatures can cast spells too (?).
+
+## Bindwood
+The rosewood is a very hard and dark reddish brown type of wood. It is hard to work and takes high polish. The people of Kestal call it _bindwood_, as he has the amazing ability to drive the energy of a source of power.
```
Problems here:
* The name of the universe is not “Kestal”
* Which people name it bindwood ? When?

Adkorh replied:
Ok about "Kestal", didn't know. We could for now say that it is commonly called _bindwood_...

kestal replied:
yeah, and choosing a name for the universe could be useful too, although the fact of not specifying one strengthens the in-universe immersion I'd say

Adkorh replied:
Right. It depends on which point of view we focus: narrative of descriptive. Although assuming they know about astrology, a generic term like _Earth_ could be necessary to compare with other planets.

kestal reviewed (commented):
I'm not sure about naming the warp "praeter". On one hand, it allows us to have our own nomenclature, and the name sort of makes sense; on the other, why wouldn't we just use *warp*, it being our original source…

Also, "out of universe" comments shouldn't be made inside the documents.

ghost commented:
I just though that because the _warp_ mentions a sort of affiliation with Chaos, it would be better to mark the difference on that point by changing name.

Bandlou committed:
Update 3. The Frame of Magic.md

Bandlou committed:
Update 2. The Art of Magic.md

Bandlou committed:
Update 1. The Praeter.md

Bandlou committed:
Update 1. The Praeter.md

Bandlou committed:
Update 1. The Praeter.md

Bandlou committed:
Update The Treaty on Magic.md

Bandlou committed:
Update The Treaty on Magic.md

People of Nendal, as well as the Southeners, are just temporary names. People of Nendal could look like Azetec tribe living in jungles or  norse tribes living in some cold and mountainous places. In any cases, they wouldn't be the most developed people in the time where Arkas rises.

Bandlou committed:
Delete The Nendal Codex.md

Transfered in the _Treaty on Magic_ file.

Bandlou committed:
Update 3. The Frame of Magic.md

Bandlou committed:
Update 3. The Frame of Magic.md

Bandlou committed:
Update 3. The Frame of Magic.md

Bandlou committed:
Update 3. The Frame of Magic.md

Bandlou committed:
Update 3. The Frame of Magic.md

Bandlou committed:
Update 3. The Frame of Magic.md

Bandlou committed:
Update 3. The Frame of Magic.md

Bandlou committed:
Update 3. The Frame of Magic.md

Bandlou committed:
Update 2. The Art of Magic.md

kestal commented:
Concerning "warp" vs "praeter", I think we'll go with "praeter".

Bandlou committed:
Update The Treaty on Magic.md

Bandlou committed:
Update The Treaty on Magic.md

The Highlands' Folk is inspired by the Tibetan monks, and I totally assume that. The point here is to develop them to make them more unique, but the basis still: pacific people, spirituality leads to magic, magic to protect the weaks.

ghost commented:
**The Treaty on Magic**
I want to explicit my vision of the folks I mentioned there. All theses folks have temporary names used to simplify the writing.

_People of Nendal_ could look like Aztec or Norse tribes (I actually would prefer this one). In any cases, they wouldn't be the most developed people in the time where Arkas rises.

The _Highlands' Folk_ is totally inspired by the native Tibetans, and it's assumed. The goal is to make them different, unique, while keeping as basis these two things: they're a pacific people, spiritual awakening leads to magic gifted from the gods to protect the weaks.

Bandlou committed:
Update The Treaty on Magic.md

Bandlou committed:
Update The Treaty on Magic.md

Bandlou committed:
Update The Treaty on Magic.md

Bandlou committed:
Update The Treaty on Magic.md

Bandlou committed:
Update Elves.md

Bandlou committed:
Update Elves.md

Bandlou committed:
Update Elves.md

ghost commented:
**Elves update**
I just made a little update concerning the elves, regrouping the ideas of the _Species?_ issue and those about magic.

Bandlou committed:
Update The Treaty on Magic.md

Bandlou committed:
Update The Treaty on Magic.md

Bandlou committed:
Update The Treaty on Magic.md

Bandlou committed:
Update The Treaty on Magic.md

Bandlou committed:
Update The Treaty on Magic.md

Bandlou committed:
Update The Treaty on Magic.md

Bandlou committed:
Update The Treaty on Magic.md

Bandlou committed:
Update The Treaty on Magic.md

Bandlou committed:
Update The Treaty on Magic.md

Bandlou committed:
Update The Treaty on Magic.md

Bandlou committed:
Update The Treaty on Magic.md

Bandlou committed:
Update The Treaty on Magic.md

Bandlou committed:
Update The Treaty on Magic.md

ghost commented:
I actually made some changes to match the Nendal's folk story with the Fragmented Realm.

lord-koko commented on Magic/1. The Praeter.md:
```diff
@@ -0,0 +1,24 @@
+## The direct way
+Accessing the praeter directly requires years and years of training and a lot of energy. Thereby practically nobody can perform magic this way. Adepts of this methode are extremely powerful.
+
+It is said that some of them could even use the praeter to keep their spirit immortal and even come back in the real world as a newborn. They are sometimes perceived as divine or even as gods by certain folks.
```
Entering the preater still sounds a bit confused to me. Should we keep it like this to make it a somehow secret technique ? Also, I am not sure to like the idea of a completely separated plan, where people enter and leave. What do you think of a more mixed with reality thing ?

lord-koko replied:
Yes, i like that way more !

lord-koko commented on Magic/1. The Praeter.md:
```diff
@@ -0,0 +1,24 @@
+
+The inert sources of power fade away as they are used and may regenerate over time. 
+
+Living sources gets exhausted when used and need time to recover, exactly like the caster. However, if pushed to much, they can also faint and even die. Nevertheless, they can defend themselves from the caster in a sort of mental duel.
```
I am not sure about this mental duel, I think they should be consummed like coal.

Adkorh replied:
Yes but the issue here is when an elf try to consume another elf...

lord-koko commented on Magic/1. The Praeter.md:
```diff
@@ -0,0 +1,24 @@
+
+Living sources gets exhausted when used and need time to recover, exactly like the caster. However, if pushed to much, they can also faint and even die. Nevertheless, they can defend themselves from the caster in a sort of mental duel.
+
+The magical beast are standing between the two dimensions, while the elves are more uniques.
```
Again, the question of two dimensions separated this strong bug me a little. it sounds like the preater has a "physical" form. I see preater as a thing where you cant "stand" or "walk into".

Adkorh replied:
Ok this wasn't again a judicious choice of words. We could say magical beasts feeling the praeter in a natural and strong way

lord-koko commented on Magic/2. The Art of Magic.md:
```diff
@@ -0,0 +1,30 @@
+The use of strong bonds sources of power requires a way more important training and is reserved for talented wizards.
+
+The strength and the length of a spell are limited by both the caster and, if used, the source of power.
+Performing magic is exhausting for the caster, as draining energy from the praeter requires a great concentration. Although it can't injure the caster physically, it still can faint him in extreme cases.
```
I think that magic could hurt the body of the caster. To discuss.

lord-koko commented on Magic/2. The Art of Magic.md:
```diff
@@ -0,0 +1,30 @@
+The strength and the length of a spell are limited by both the caster and, if used, the source of power.
+Performing magic is exhausting for the caster, as draining energy from the praeter requires a great concentration. Although it can't injure the caster physically, it still can faint him in extreme cases.
+
+Casting does not require any gesture, lyrics or ritual.
```
Really ?

lord-koko commented on Magic/2. The Art of Magic.md:
```diff
@@ -0,0 +1,30 @@
+Casting does not require any gesture, lyrics or ritual.
+
+## The sources of power
+All the sources of powers allow the caster to perform any kind of magic. There are two things to mention though: first each kind of spells need specific trainings to master, and secondly each kind of magic requires a specific environment to be performed.
```
Not sure about the "kind of magic" thing. What do you mean precisely. You can throw fire only from a volcano ? And I dont like the idea of different kind of magic.

Adkorh replied:
I detailed in the third part [link](https://github.com/Adkorh/univers/blob/patch-1/Magic/3.%20The%20Frame%20of%20Magic.md)

lord-koko commented on Magic/The Treaty on Magic.md:
```diff
@@ -0,0 +1,31 @@
+The _Treaty on Magic_ is a series of books about magic divided in multiple volumes. There is no unique writer but the first volume was mainly achieved by the wizard Brynjar. If he was a wizard, he was first a scholar, collecting as much informations he could about magic and the way people see it in the world.
+
+## Volume I: Generalities
+> There is no school to learn magic, and there are no many wizards too. I think most of those seeking for power aren't patient enough to spend years learning the ways of the praeter. Becoming a wizard is also a lonely path for the most, and requires a great discipline too. [...] This is an hard exercice to feel the praeter in general. Even in places of power where the task is eased, you will have ...
```
I like this!

lord-koko commented on Magic/The Treaty on Magic.md:
```diff
@@ -0,0 +1,31 @@
+## Volume I: Generalities
+> There is no school to learn magic, and there are no many wizards too. I think most of those seeking for power aren't patient enough to spend years learning the ways of the praeter. Becoming a wizard is also a lonely path for the most, and requires a great discipline too. [...] This is an hard exercice to feel the praeter in general. Even in places of power where the task is eased, you will have ...
+
+> In our way of learning magic, the first step for the apprentice is to go in pilgrimage to a common magical place. There he must show patience, discipline and spirit opening to start feeling the praeter. This first step is usually hard and very shattering once reached. In the beginning, the time spent in the praeter is very tiring and long breaks are required between each trip. At this point of  ...
```
Problem with "kind of magic" already told earlier. We could just say that some spells or magical things are easier to do than others.

lord-koko commented on Magic/The Treaty on Magic.md:
```diff
@@ -0,0 +1,31 @@
+> The people of Nendal actually believe that Arkas and the other witchers are gods that initially live in the praeter. They think that some of them decided to come on earth and, as they are gods, their soul can't die and they can come back as newborns as they please. What is actually funny is that most of their witchers believe they are gods, simply assuming that after many ressurections, they ju ...
+
+> At the time I'm writing this, the Nendal's folk is ruled by a theocratic government with Arkas as the spiritual leader, and where the revered witchers occupy high functions. The people are considered inferiors to the witchers and the king. Arkas even adopt the title _King of Gods_. When he reborns and is still in young age, a provisional council is formed to rule. The witchers are very heteroge ...
+
```
I really like this nendal story. Resurection is a powerfull thing, it think that Arkas should be one of the only to have performed this.

lord-koko commented on Magic/The Treaty on Magic.md:
```diff
@@ -0,0 +1,31 @@
+## Volume I: The Highlands' Folk
+> There is also this pacific folk living in the Highlands. They assume that the contact with the praeter represent a way to their God. They actually believe that the spiritual awakening leads to the praeter, and that those who reached it are gracefully gifted with magic. Their beliefs mention that this gift comes with the mission to protect the weaks. Thus the wizards there are called _Monks_ and ...
+
+> Unlike our wizards, who mostly act seperatly, their monks have built temples in the surrounding area of strong magical places and live there, secluded in hierarchized communities. We actually don't know their position concerning the practice of ressurection.
```
Like the monks

Bandlou committed:
Update The Treaty on Magic.md

kestal commented:
@Adkorh: I'm happy you're working so tirelessly on expanding our universe!
I've added you to the contributors: That means you can directly make any change you want to the repo, without having to go through the fork->pull request procedure.
In my opinion, it would be best that only small changes (typos, etc) are made directly; any non trivial change should still be made as a pull request, so that everyone can comment on it and agree (or not!) to merge it.

I'm not sure where precisely to make the following remarks, so here they go:
* I don't think the world we have here is spherical, so having "arctic circle" doesn't really make sense for now. I personally like the idea of a flat world, maybe even infinite. This should maybe be discussed in a brand new issue?
* I think it would be better for the new societies you created to lie on continent II. The problem we have here is that the culture you wrote are heavily influenced by praeter, while everything on continent I is pretty much non-magical. One way to explain that would be that the "magical currents" are stronger on continent II, so that placing your folks there is the most reasonable explanation I have so far.

ghost commented:
Thanks, i agree with the pull request for important changes.

- Ok i'm fine with the flat world idea, but the daylight changing with season is important and even if the world isn't spherical, I think it should be possible to play with this
- Ok didn't know about that, we could just put them in continent II in that case, modifying a bit the story but that wouldn't make big changes
